### PR TITLE
Fix formatting to use %n in place of \n

### PR DESCRIPTION
**Pull request by ajaygeorge**
%n is a platform-independent newline character which is to be
preferred in place of \n for formatting.

Test plan - Unit tests

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by ajaygeorge** [comment]
native-specific tests are failing due to an un-related issue.

**Review by rschlussel — APPROVED** [review]

**Comment by rschlussel** [comment]
Merging is blocked still.  It needs review from a cpp code owner since it touches a file in presto-native.

**Review by majetideepak — APPROVED** [review]
+1 for presto-native changes.

**Pull request merged**
